conjur/cli.py

```diff
@@ -183,10 +196,11 @@ def run(self, *args):
 
         resource, args = Cli._parse_args(parser)
 
-        # TODO Add tests for exception handling logic
         # pylint: disable=broad-except
         try:
             Cli.run_client_action(resource, args)
+        except KeyboardInterrupt:
+            sys.exit(0)
```

Comment:
suggestion: add debug log?

Reply:
Not critical. I think the user seeing their `^C` is enough